app/scripts/app-init.js

```diff
@@ -30,6 +32,8 @@ function importAllScripts() {
     tryImport('./lockdown-run.js');
     tryImport('./runtime-cjs.js');
   }
+  // eslint-disable-next-line
+  self.scriptLoaded = true;
```

Comment:
hmmm I didn't know you could write to an arbitrary property to `self` 🤔

Reply:
I found, it does works :)